cwf/gateway/configs/pipelined.yml

```diff
@@ -157,8 +157,7 @@ has_quota_port: 51115
 no_quota_port: 51125
 
 # For 5G Functionality Support flag
-5G_feature_set:
- enable: false
```

Comment:
is there reason for changing the nested attribute type here?

Reply:
Yeah to make it uniform across all modules we made it independent bool attribute by directly assigning the value. So not added one more attribute as nested[as it was in previous code].